examples/jsm/loaders/GLTFLoader.js

```diff
@@ -3615,7 +3626,9 @@ class GLTFParser {
 							// representing inTangent, splineVertex, and outTangent. As a result, track.getValueSize()
 							// must be divided by three to get the interpolant's sampleSize argument.
 
-							return new GLTFCubicSplineInterpolant( this.times, this.values, this.getValueSize() / 3, result );
+							const normalizeQuaternion = PATH_PROPERTIES[ target.path ] === PATH_PROPERTIES.rotation;
```

Comment:
It would be nice if the interpolant didn't depend on local scope of the loop. It's the kind of thing that can behave weirdly with 'var', ought to work with 'const', but still feels better just to avoid. Instead I think we could do:

```js
const normalizeQuaternion = this instanceof QuaternionKeyframeTrack;
```